### PR TITLE
Update the welcome page to create a temporary admin user

### LINKED ISSUE #30010: Update the welcome page to create a temporary admin user
The admin users created through the bootstrapping process as specified in https://github.com/keycloak/keycloak/issues/9829 will be handled as temporary.

The welcome page allows creating an initial admin user already. This issue covers marking such user created this way  as temporary.

### PR DESCRIPTION
**Pull request by mabartos**
Closes #30010

image

### CODE REVIEW

**Review by vmuzikar — DISMISSED** [review]
LGTM from the Cloud Native perspective. Thanks @mabartos

**Comment by mabartos** [comment]
Not sure what might be the cause of these failures as I wasn't able to reproduce it. Rebased

**Review by vmuzikar — DISMISSED** [review]

**Comment by mabartos** [comment]
@keycloak/ui-maintainers Could you please check it? Thanks!

**Comment by mabartos** [comment]
@ssilvert @edewit Hi, are these changes okay to you? Thanks

File: themes/src/main/resources/theme/keycloak/welcome/index.ftl [hunk 1]
```diff
@@ -28,13 +28,13 @@
           <main class="pf-v5-c-login__main">
             <header class="pf-v5-c-login__main-header">
               <#if localUser>
-                <h1 class="pf-v5-c-title pf-m-3xl">Create an administrative user</h1>
+                <h1 class="pf-v5-c-title pf-m-2xl">Create a temporary administrative user</h1>
                 <#if !successMessage?has_content>
-                  <p class="pf-v5-c-login__main-header-desc">To get started with ${productName}, you first create an administrative user.</p>
+                  <p class="pf-v5-c-login__main-header-desc">To get started with ${productName}, you first create a temporary administrative user. To harden security, create a permanent one and delete the temporary one.</p>
```
Comment (ssilvert) [inline themes/src/main/resources/theme/keycloak/welcome/index.ftl:33]: I think this makes it a little more clear that the permanent user should be created later:
```suggestion
                  <p class="pf-v5-c-login__main-header-desc">To get started with ${productName}, you first create a temporary administrative user.  Later, to harden security, create a new permanent administrative user and delete the temporary user that was created during this setup.</p>
```

**Review by ssilvert — DISMISSED** [review]

File: themes/src/main/resources/theme/keycloak/welcome/index.ftl (hunk 1 above)
Comment (mabartos) [reply themes/src/main/resources/theme/keycloak/welcome/index.ftl]: @ssilvert LGTM, thanks!

**Comment by mabartos** [comment]
@ssilvert @vmuzikar Could you please review it again? Thanks!

**Review by vmuzikar — APPROVED** [review]
Still LGTM!

**Review by jonkoops — APPROVED** [review]
LGTM

**Comment by mabartos** [comment]
Thanks for the reviews!

@vmuzikar I think it should be ready to merge.

**Pull request merged**